plugin-volume/volumepopup.h

```diff
@@ -81,6 +82,7 @@ private slots:
     QPoint m_pos;
     Qt::Corner m_anchor;
     AudioDevice *m_device;
+    QTimer *mWheelTimer; // for showing tooltip with high-resolution mice
```

Comment:
If we are adding a member to the class, we could add also the "_delta", and remove the static variable inside the method `handleWheelEvent()`

Reply:
→ https://github.com/lxqt/lxqt-panel/pull/1856#issuecomment-1380217643 → first paragraph

→ https://github.com/lxqt/lxqt-panel/pull/1856#issuecomment-1381416968